util.go

```diff
@@ -276,9 +278,26 @@ func makenewwindow(t *Text) *Window {
 	return w
 }
 
-func mousescrollsize(nl int) int {
-	// Unimpl()
-	return 1
+func mousescrollsize(maxlines int) int {
```

Comment:
This function is invoked for every key press that should scroll the Window. Perhaps it would be nicer to parse the environment variable only once and cache that?

Reply:
Yep, I was trying to avoid introducing more global variables. Maybe move this into a new `internal/drawutil` package?

Reply:
I've moved it to `internal/drawutil` package.